assets/js/modules/analytics/common/account-create.js

```diff
@@ -123,12 +129,23 @@ export default function AccountCreate() {
 				{ __( 'You will be redirected to Google Analytics to accept the Terms of Service and create your new account.', 'google-site-kit' ) }
 			</p>
 
-			<Button
-				disabled={ ! canSubmitAccountCreate }
-				onClick={ handleSubmit }
-			>
-				{ __( 'Create Account', 'google-site-kit' ) }
-			</Button>
+			<div className="googlesitekit-setup-module__action">
+				<Button
+					disabled={ ! canSubmitAccountCreate }
+					onClick={ handleSubmit }
+				>
+					{ __( 'Create Account', 'google-site-kit' ) }
+				</Button>
+
+				{ ( accounts && !! accounts.length ) && (
```

Comment:
Probably okay here, but this may result in the Back button suddenly appearing and possibly not being there right away although it should be. I wonder if it would be a cleaner approach to catch `accounts === undefined` in the if clause further above, returning the `ProgressBar` when accounts aren't loaded yet.

Reply:
That shouldn't ever happen because that logic is already handled by `SettingsEdit`. I suppose it wouldn't hurt to add though.

Reply:
Yeah I'd prefer for components to not rely on parent components where avoidable.